public/js/charts.js

```diff
@@ -62,7 +61,7 @@ export default class LighthouseChart {
       return;
     }
     const pagewith = Math.max(document.documentElement.clientWidth, window.innerWidth || 0);
-    fetch(CHART_BASE_URL + pwaId)
+    fetch(this.baseUrl + pwaId + '?graph=true')
```

Comment:
The chart should receive the whole URL, and not have to worry about assembling it. 

If we want to be more generic,  we could pass a datasource instead of a Url. 

```
class UrlDatasource {
  contructor(url) {
    this._url = url;
  }
  
  get() {
    return fetch(this._url);
  }
}
```
and then do:

```
datasource.get().then(...)
```
I'd imagine something like this if we decide to pass the data inside a `script type=application/json` section to avoid another request. Might be an overkill tough